hbt/config/hist_hooks.py

```diff
@@ -187,6 +187,278 @@ def qcd_estimation(task, hists):
 
         return hists
 
+    def flat_s(task, hists: dict[hist.Histogram]) -> dict[hist.Histogram]:
```

Comment:
`dict[hist.Histogram]` only refers to the dict values so it's invalid.

Reply:
done